config/config.go

```diff
@@ -82,6 +104,24 @@ func Parse(cfg map[string]string) (Config, error) {
 		config.TabletType = strings.ToLower(tabletType)
 	}
 
+	if maxRetriesStr := cfg[KeyMaxRetries]; maxRetriesStr != "" {
+		retries, err := strconv.Atoi(maxRetriesStr)
+		if err != nil {
+			return Config{}, fmt.Errorf("invalid retries: %w", err)
+		}
+
+		config.MaxRetries = retries
+	}
+
+	if retriesTimeoutStr := cfg[KeyRetryTimeout]; retriesTimeoutStr != "" {
+		retriesTimeout, err := strconv.Atoi(retriesTimeoutStr)
```

Comment:
Let's better use the `time.ParseDuration` function. Thus we can parse different durations, like `10ms`, `1s`, `2m`, etc.